iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothubservices/TransportClientTests.java

```diff
@@ -340,7 +340,7 @@ public void sendMessagesOverAmqpsWsMultithreaded() throws InterruptedException,
                             deviceListAmqps[i], clientArrayList.get(i), NUM_MESSAGES_PER_CONNECTION, NUM_KEYS_PER_MESSAGE, SEND_TIMEOUT_MILLISECONDS, cdl))
                     .start();
         }
-        cdl.await();
+        cdl.await(3, TimeUnit.MINUTES);
```

Comment:
>await [](start = 12, length = 5)

These tests aren't failing at this point, but if they ever did, they would run for our entire 15 minute test timeout which is unacceptable.